front/app/containers/ProjectsShowPage/shared/survey/SmartSurvey.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const StyledIframe = styled.iframe`
+  display: block;
+  border: none;
+  height: 600px;
+  flex-basis: 640px;
+  border: 1px solid #ccc;
```

Comment:
Why does it contain the border property twice?

Reply:
By mistake! 😓